recipes/xraylib/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "3.1.0" %}
+
+package:
+  name: xraylib
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/tschoonj/xraylib
+  git_rev: xraylib-{{ version }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy
```

Comment:
Ok according to the Cython section of the numpy 1.10.1 docs:

```
This module shows use of the cimport statement to load the definitions from the numpy.pxd header that ships with Cython. It looks like NumPy is imported twice; cimport only makes the NumPy C-API available, 
```

Given that, I think it should be assumed that this library *is* using the numpy C api and so 'numpy x.x' should be in the recipe